docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -403,21 +403,28 @@ __Example 1__: Similar to the `/genes` endpoint, a user would like to retrieve i
 === "Filter"
 
     ```json
-     {
-       "op":"in",
-       "content":{
-          "field":"cosmic_id",
-          "value":[
-             "COSM1135366"
-          ]
-       }
```

Comment:
this worked fine for me, btw, when encoded. 

I don't know if it's standard to include the top-level "filters" section when passing the json to the filters param. I don't think we do that elsewhere in our docs, but I didn't check thoroughly.